src/types/AST.ts

```diff
@@ -23,15 +24,21 @@ export type AST =
 
 export interface AbstractAST {
   comment?: string
+  jsdoc?: Record<string, any>
```

Comment:
Nit: Please keep capitalization consistent, and avoid `any` like fire.

```suggestion
  jsDoc?: {[k: string]: unknown}
```